apps/webClient/src/views/About/index.tsx

```diff
@@ -0,0 +1,143 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { makeStyles } from "@material-ui/styles";
+import Grid from "@material-ui/core/Grid";
+import Box from "@material-ui/core/Box";
+import FlareIcon from "@material-ui/icons/Flare";
+import Typography from "@material-ui/core/Typography";
+import { Theme } from "@material-ui/core/styles/createMuiTheme";
+import Logo from "../../components/Logo/";
+import NavButton from "../../components/NavButton";
+
+interface Props {}
+
+const useStyles = makeStyles((theme: Theme) => ({
+  contrastHeader: {
+    color: "white",
+    textTransform: "uppercase",
+  },
+}));
+
+const About: React.FC<Props> = ({}) => {
+  const classes = useStyles();
+
+  return (
+    <Box pt={16} height="100vh">
```

Comment:
We had a `root` style in `component/View` set to 100vh. The `/our-work` page, goes beyond the viewport, so I had to change it to 100%. However, now in About, Home, ect, if there isnt enough content to fill the viewport, the background color cuts off where the content ends.
![Screenshot from 2020-03-10 13-24-34](https://user-images.githubusercontent.com/16766024/76357073-63d69580-62ed-11ea-8a98-3f1651307920.png)

My solution is to add `100vh` per page that needs it. Let me know if there is a better way of doing this, this certainty doesn't feel elegant

Reply:
Glad you recognize the inelegance.  I think we can fix this based on my comment above.